muscle3/muscle_manager.py

```diff
@@ -109,10 +110,20 @@ def manage_simulation(
     success = manager.wait()
 
     if not success:
+        log_file = run_dir_path / 'muscle3_manager.log'
+
+        print()
         print('An error occurred during execution, and the simulation was')
         print('shut down. The manager log should tell you what happened.')
-        print('You can find it at')
-        print('   ', run_dir_path / 'muscle3_manager.log')
+        print('Here are the final lines of the manager log:')
+        print()
+        print('-' * 80)
+        print(last_lines(log_file, 30), '    ')
```

Comment:
Can we maybe silence components with ConnectionLost errors in this output? They might prevent you from seeing the actual program that had a problem.

Reply:
Tricky, the output looks different depending on whether it's a Python or a native component, and then there could be false positives, and the user could catch the exception and print something else, and...

We could do something more heavy-duty, but that would be a separate PR, and I cannot think of a way to do it that I actually like. To be continued in https://github.com/multiscale/muscle3/issues/224